docs/2.8. Usage - Testing Controllers.md

```diff
@@ -0,0 +1,19 @@
+# Testing Controllers
+
+## Running the Semi-automatic Controller Tests
+
+Launch the Testing Façade from the testingfacade directory. By default the Testing Façade will run on port 5001
```

Comment:
Where is it described how to change the default port? If it's just another a _Config.py_ setting, then it's probably worth a link to the doc about _UserConfig.py_ and naming the variable here?